File: TI1.3 REST_JAVA_CONUNI_GRP3/01 SERVIDOR/src/main/java/ec/edu/espe/conversionapi/enums/TemperatureUnit.java
package ec.edu.espe.conversionapi.enums;

/**
 * Unidades de temperatura
 */
public enum TemperatureUnit {
    CELSIUS("Celsius"),
    FAHRENHEIT("Fahrenheit"),
    KELVIN("Kelvin");

    private final String displayName;

    TemperatureUnit(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: TI1.4 REST_DOTNET_CONUNI_GRP3/02 CLIMOV/convrtfull/app/src/main/java/ec/edu/espe/conv_rtfull/data/model/TemperatureUnit.kt
package ec.edu.espe.conv_rtfull.data.model

/**
 * Unidades de temperatura
 */
enum class TemperatureUnit(val displayName: String, val code: Int) {
    CELSIUS("Celsius", 0),
    FAHRENHEIT("Fahrenheit", 1),
    KELVIN("Kelvin", 2),
    RANKINE("Rankine", 3)
}
